plugins/nf-amazon/src/main/nextflow/cloud/aws/batch/AwsBatchFileCopyStrategy.groovy

```diff
@@ -154,7 +154,7 @@ class AwsBatchFileCopyStrategy extends SimpleFileCopyStrategy {
     }
 
     static String uploadCmd( String source, Path target ) {
-        "nxf_s3_upload '$source' s3:/$target"
+        "nxf_s3_upload \"$source\" s3:/$target"
```

Comment:
```suggestion
        "nxf_s3_upload ${Escape.path(source)} s3:/${Escape.path(target)}"
```

Good finding. Likely the best solution is to follow the implementation of 

https://github.com/nextflow-io/nextflow/blob/287e23b5dfaf2262255d9c41b88b9704558d56f4/plugins/nf-amazon/src/main/nextflow/cloud/aws/batch/AwsBatchFileCopyStrategy.groovy#L152-L154

Reply:
Done